inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/dialect/ClickHouseDialect.java

```diff
@@ -154,6 +178,67 @@ public String getDeleteStatement(String tableName, String[] conditionFields) {
                 Arrays.stream(conditionFields)
                         .map(f -> format("%s = :%s", quoteIdentifier(f), f))
                         .collect(Collectors.joining(" AND "));
-        return "ALTER TABLE " + quoteIdentifier(tableName) + " DELETE WHERE " + conditionClause;
+        Pair<String, String> pair = getDatabaseAndTableName(tableName);
```

Comment:
use a name that illustrate the actual data, since "pair" can be infered from its class type

Reply:
ok